src/pages/api/ontology/index.js

```diff
@@ -1,12 +1,17 @@
 import path from 'path'
 import {promises as fs} from 'fs'
 import log from 'loglevel'
-import { getUbkgCodes } from '../../../config/config'
+import {getRootURL, getUbkgCodes} from '../../../config/config'
+import {get_auth_header, get_headers_from_req, get_json_header} from "../../../lib/services";
 
 const ONTOLOGY_CACHE_PATH = path.join(process.cwd(), 'cache')
 export default async function handler(req, res) {
+    let auth = get_auth_header({req, res})
+    let headers = get_json_header(get_headers_from_req(req.headers, auth))
+    let response  = await fetch(`${getRootURL()}api/auth/token`, {method:'GET', headers})
```

Comment:
I think we want to call `GET /privs/has-data-admin` in ingest-api here. The portal's `/api/auth/token` ultimately calls Globus's `/v2/oauth2/token/introspect` endpoint which checks if the token is valid. We should check if the token belongs to an admin user.

Reply:
Thanks much @tjmadonna for looking at this. Updated.